File: 0009-palindrome-number/0009-palindrome-number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        stringX = str(x)
		#reverse the string by the last index from backwards to forwards
        reverseX = stringX[::-1]
		#if the forward version of the string equals the reversed then it's a palindrome
        if stringX == reverseX:
            return(True)
        return(False)
        
       

File: 0009-palindrome-number/0009-palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {
        if (x< 0){
            return false;
        }
        long rev=0;
        long temp=x;
        while( temp>0){
            long ld=temp%10;
            rev=rev*10+ld;
            temp=temp/10;
        }
        return rev==x;
    }
}
